docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,18 @@
 
 ## January 2024
 
+**31 January 2024**
+
+Frontend version: 1.0.0-20240125.1442 | Backend version: 1.88.3-20240131.1209
+
+| Type   | Change  | Description |
+| --- | --- | --- |
+| **Enhancement**   | New `send targeted email` Automation API | This change the ability to send targeted emails with recipients specified in the 'To' and 'Cc' fields, suitable for direct interactions like approval processes. This feature is distinct from email blasts, which uses 'Bcc' for privacy. |
+| **Enhancement** | API performance enhancement  | Updated `Get/List Users` APIs to exclude `seedStatus` and `seedDevices` by default, improving response times. This change, effective for three months, ensures backward compatibility by including seed information irrespective of the `seedInfo` query parameter during this transitional period. |
```

Comment:
Can we indicate which day this special case will expire?

Reply:
@kpslp can mention backward compatibility will expire 24 Apr (stg) & 1 May (prod)

Reply:
done